tools/export/mcuxpresso/empty_cproject.tmpl

```diff
@@ -0,0 +1,3 @@
+{% extends "mcuxpresso/.cproject.tmpl" %}
+
+{% block cpu_config %}{% endblock %}
```

Comment:
A quick grep indicates that this template is not directly referenced, and since we don't have a target named "empty", I wonder how this exporter uses it.